aws_lambda/requirements.txt

```diff
@@ -1,4 +1,16 @@
+boto3==1.34.123
+botocore==1.34.145
+certifi==2024.7.4
+cffi==1.16.0
+charset-normalizer==3.3.2
+cryptography==43.0.0
+github-api-toolkit @ git+https://github.com/ONS-Innovation/github-api-package.git
```

Comment:
Will want to pin to a release (separate JIRA to cover this)

Reply:
Jira ticket added, will amend next sprint